src/test/java/ParserTest.java

```diff
@@ -0,0 +1,21 @@
+import duke.DukeException;
+import duke.Parser;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ParserTest {
+    @Test
+    public void parseBye() throws DukeException {
```

Comment:
Perhaps you can consider naming your tests in the format `featureUnderTest_testScenario_expectedBehavior()`?